src/test/java/net/lingala/zip4j/crypto/PBKDF2/BinToolsTest.java

```diff
@@ -0,0 +1,65 @@
+package net.lingala.zip4j.crypto.PBKDF2;
+
+import java.lang.reflect.Array;
+import net.lingala.zip4j.crypto.PBKDF2.BinTools;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+
+public class BinToolsTest {
+  @Rule
+  public final Timeout globalTimeout = new Timeout(10000);
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
```

Comment:
`ExpectedException` is a class not a variable

Reply:
I meant to change the variable name `thrown` to `expectedException`. This naming convention has been followed in other tests across this library. This will keep the naming convention consistent.